### PR TITLE
Test Results Management for Travis CI

### PR DESCRIPTION
**Pull request by munderseth**
We’ve made a few minor changes to demonstrate how [Testspace.com](https://www.testspace.com/) can be used by your team to better manage your **test results**. Note that there is **no** impact on the workflow and the product is **completely free** for open source.

YOUR TEST RESULTS from `our fork` at https://open.testspace.com/projects/TryTestspace:react-dates.

Testspace Badge:

[Space Health](https://open.testspace.com/spaces/122486?utm_campaign=badge&utm_medium=referral&utm_source=test "Test Cases")

----

We use Testspace for our own development along with `GitHub` and `Circle CI`. To see how Testspace really works, check out our demo.

**LIVE DEMO**: https://demo.testspace.com. 

#### Setup is Easy

1. Create a free [Open Source Account](https://signup.testspace.com/?plan_id=open.2)
2. Create a **New Project** from the list of Github repo's
3. Add a Travis **Environment Variable**: Name: `TS_ORG`  Value: `organization name` - based on name selected in step 1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +38,7 @@ env:
     - REACT=16
 sudo: false
 matrix:
-  fast_finish: true
+  fast_finish: false
```
Comment (ljharb) [inline .travis.yml]: why would we want to disable this? That will slow down our builds a lot.

File: .travis.yml [hunk 2]
```diff
@@ -17,7 +20,15 @@ script:
   - 'if [ -n "${BUILD-}" ]; then npm run lint ; fi'
   - 'if [ "${TEST-}" = true ]; then npm run tests-only ; fi'
   - 'if [ -n "${KARMA-}" ]; then npm run tests-karma ; fi'
-  - 'if [ -n "${COVERAGE-}" ] && [ "${TRAVIS_BRANCH-}" = "master" ]; then npm run cover; cat ./coverage/lcov.info | ./node_modules/.bin/coveralls ; fi'
+  - 'if [ -n "${COVERAGE-}" ]; then npm run cover; cat ./coverage/lcov.info | ./node_modules/.bin/coveralls ; fi'
```
Comment (ljharb) [inline .travis.yml]: The master branch restriction needs to remain applying to the coveralls binary command.

File: .travis.yml (hunk 1 above)
Comment (munderseth) [reply .travis.yml]: * The overall results (including passing) represent the overall `Test Design`, we present it based on the repo source organization. It is also used for Metric purposes, related to failures and resolution rates. Can take a look at our stuff using the `demo` link above. We generate a number of metrics based on the continuous collection of CI results. 

* I believe the `fast_finish` set to `false` is a mistake on our side

File: .travis.yml (hunk 2 above)
Comment (munderseth) [reply .travis.yml]: **Agree**. Was changed it only so we could leverage the Coverage using our branch with our Travis account -- show how the results look in Testspace.

**Comment by ljharb** [comment]
We no longer use travis, and it's been awhile.

**Pull request closed without merging**